apis/components/v1alpha1/datasciencepipelines/types.go

```diff
@@ -0,0 +1,21 @@
+// +kubebuilder:object:generate=true
+
+// Package datasciencepipelines provides a set of types used for DataSciencePipelines component
+package datasciencepipelines
+
+import operatorv1 "github.com/openshift/api/operator/v1"
+
+type PreloadedPipelinesSpec struct {
+	InstructLab InstructLabPipelineSpec `json:"instructLab,omitempty"`
```

Comment:
Can we get something like the following comment: 

```suggestion
        //  Configure whether to auto import the InstructLab pipeline pon any new pipeline server (or DSPA) creation. 
        // Please note that the pipeline requires that trainingoperator be enabled in the DSC to run.
	InstructLab InstructLabPipelineSpec `json:"instructLab,omitempty"`
```

Reply:
done